src/test/scala/chiselTests/MixedVecSpec.scala

```diff
@@ -62,7 +62,7 @@ class MixedVecZeroEntryTester extends BasicTester {
     val bar = zeroEntryMixedVec
   }
   require(0.U.asTypeOf(bundleWithZeroEntryVec).getWidth == 1)
-  require(bundleWithZeroEntryVec.asUInt.getWidth == 1)
+  require(WireInit(bundleWithZeroEntryVec, DontCare).asUInt.getWidth == 1)
```

Comment:
why is this different than the one in Vec.scala? 
```
require(bundleWithZeroEntryVec.asUInt.getWidth == 1)
```

Reply:
I changed them several days apart and forgot 🙂. I'll make this one consistent with the one in `Vec.scala`.

Reply:
Fixed.